docs/user-guide/options.md

```diff
@@ -70,7 +70,7 @@ This option should be a string that resolves to a JS module that exports a [Post
 
 If you want to lint two or more different languages, you can combine `customSyntax` with the [`overrides`](./configure.md#overrides) configuration property.
 
-Using the Node.js API, the `customSyntax` option can also accept a [Syntax object](https://postcss.org/docs/how-to-write-custom-syntax#syntax). Stylelint treats the `parse` property as a required value.
+Using the Node.js API, the `customSyntax` option can also accept a [Syntax object](https://postcss.org/docs/how-to-write-custom-syntax#syntax). Stylelint treats the `parse` property as a required value. Also, Stylelint supports syntax modules written in ESM.
```

Comment:
> Stylelint treats the `parse` property as a required value

Add `in that case`?

Reply:
Hum, adding `in that case` might confuse, instead. But I'm open to a better description.

Of course, we can improve it later.